osu.Game/Overlays/Direct/DirectPanel.cs

```diff
@@ -34,6 +34,7 @@ public abstract class DirectPanel : Container
 
         public PreviewTrack Preview => PlayButton.Preview;
         public Bindable<bool> PreviewPlaying => PlayButton.Playing;
+        public abstract DownloadButton DownloadButton { get; }
```

Comment:
is this just for tests? if so, please make it protected and make a local class in the test to expose it as necessary.

Reply:
there are 2 fields other than that, I modified their accessibility for tests, should I apply the changes for them as well?

Reply:
public shouldn’t be used only for testing purposes, yes

Reply:
well, i kept `Header.DownloadButtonsContainer` because it can't be exposed